hub/api/config/db-migration/15-db-cronjob.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: hub-sync-db
+  namespace: tekton-hub-preview
+  labels:
+    app: api
+spec:
+  schedule: "*/10 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: sync-db
+              image: byrnedo/alpine-curl:0.1.8
+              args:
```

Comment:
this is going to a lot more legible if we used `[ ... ]`
e.g.: `[ -s, -q, -X, POST, api/resources/sync ]` 
We can directly access the service instead of the ingress, can't we?